kafka/init.lua

```diff
@@ -471,4 +471,5 @@ return {
     Consumer = Consumer,
     Producer = Producer,
     _LIBRDKAFKA = tnt_kafka.librdkafka_version(),
+    _VERSION = require('kafka.version'),
```

Comment:
You're missing `kafka/CMakeLists.txt` update
```bash
tt rocks STATIC_BUILD=ON make

tree .rocks/
.rocks/
├── lib
│   └── tarantool
│       └── kafka
│           └── tntkafka.so
└── share
    └── tarantool
        ├── kafka
        │   └── init.lua
        └── rocks
            ├── kafka
            │   └── scm-1
            │       ├── doc
            │       │   ├── LICENSE
            │       │   └── README.md
            │       ├── kafka-scm-1.rockspec
            │       └── rock_manifest
            └── manifest

10 directories, 7 files
```

Reply:
Fixed